agent/yadro/sensors.cpp

```diff
@@ -93,15 +91,14 @@ struct Sensor
      * @brief Object contructor.
      */
     Sensor(const std::string& folder, const std::string& name) :
-        phosphor::snmp::data::table::Item<int64_t, int64_t, bool, int64_t, bool,
-                                          int64_t, bool, int64_t, bool,
-                                          int64_t>(folder, name,
-                                                   0,        // Value
-                                                   0, false, // WarningLow
-                                                   0, false, // WarningHigh
-                                                   0, false, // CriticalLow
-                                                   0, false, // CriticalHigh
-                                                   1)        // Scale
+        phosphor::snmp::data::table::Item<double, double, bool, double, bool,
```

Comment:
Don't we need to support old platforms? How does it work now?

Reply:
Yes, exactly so.